docs/index.md

```diff
@@ -13,6 +13,7 @@ slug: documentation
 1. [Building KLEE (LLVM 3.4) using Autoconf]({{site.baseurl}}/build-llvm34-autoconf/): Instructions on how to build KLEE from source using LLVM 3.4 and KLEE's older Autoconf/Makefile based build system.
 1. [Building KLEE (LLVM 2.9)]({{site.baseurl}}/build-llvm29/): Instructions on how to build KLEE from source using LLVM 2.9.
 1. [Building STP]({{site.baseurl}}/build-stp/): Instructions on how to build STP, KLEE's default constraint solver.
+1. [Solver Pipeline]({{site.baseurl}}/docs/solver-pipeline/): Overview of the Solver pipeline and related command line arguments.
```

Comment:
The document doesn't just describe the command line option. It is also trying to describe how the solver chain works and what the idea behind each solver is.